cmd/logger/target/http/http.go

```diff
@@ -101,3 +104,9 @@ func (h *Target) Send(entry interface{}) error {
 
 	return nil
 }
+
+// MustLog returns true if logType matches log type configured
+// for this logging endpoint.
+func (h *Target) MustLog(logType string) bool {
```

Comment:
There is no reason to expose this function IMO since this usage should be internal @poornas

Reply:
t.Send() should internally use this as its internal implementation. Is there a good reason to expose this?

Reply:
alternative would be to change Send(interface{}) -> Send(interface{}, logType string) so that each target type can decide whether the message should be sent to it.